app.rb

```diff
@@ -31,19 +32,74 @@
 config_overrides[:port] = ENV['APRUVE_PORT'] unless ENV['APRUVE_PORT'].nil?
 Apruve.configure(ENV['APRUVE_API_KEY'], apruve_environment, config_overrides)
 merchant_id = ENV['APRUVE_MERCHANT_ID']
+corporate_account_id = ENV['APRUVE_CORPORATE_ACCOUNT_ID']
 
 before '/webhook_notify' do
   request.body.rewind
   @webhook_body = request.body.read
 end
 
+get '/offline_order' do
+  erb :offline_orders
+end
+
+get '/customers' do
+  content_type :json
+  send_file File.expand_path('customers.json', settings.public_folder)
```

Comment:
I think it makes more sense to store the email addresses in the json file as you are today, but leave out the shopper_id. Then you can use the existing API which allows you to look up a corporate account by email address. If you do it that way it will work in any environment.